### ISSUE TITLE
Need to add versions of minimum required dependencies to external docs

### ISSUE THREAD

**Issue by awlauria** [open-mpi/ompi#10345]
Right now the current required dependencies are listed here:  https://docs.open-mpi.org/en/v5.0.x/installing-open-mpi/required-support-libraries.html

however, minimum supported versions are not listed. A table could be utilized for this if it makes it easier to read.

**Comment by jsquyres** [comment]
@awlauria volunteered for this issue today, so I assigned it to him. 😄 

Also, since there is likely to be confusion about this kind of stuff, I marked this as a blocker.

**NOTE:** If we want to have tech enforcement of these versions, it might be a good idea to have the minimum versions in a separate data file somewhere in the repo, and let everything read the versions from that file.  For example:

* autogen (if relevant)
* configure
* sphinx (i.e., the docs)

Meaning: there would only be a single place to update a minimum version of something (e.g., PMIx), and then it automatically propagates out to all relevant places (including the docs).
